ensembl_bugs/models.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
Same her for moving this en ensembl_production app, we may need it for other services at some point